co-log/src/Colog/Message.hs

```diff
@@ -227,19 +227,19 @@ fmtRichMessageDefault RichMessage{..} = do
     maybeUtcTime  <- extractField $ TM.lookup @"utcTime"  richMessageMap
     pure $ formatRichMessage maybeThreadId maybeUtcTime richMessageMsg
   where
-    formatRichMessage :: Maybe ThreadId -> Maybe UTCTime -> Message -> Text
+    formatRichMessage :: Maybe ThreadId -> Maybe C.Time -> Message -> Text
     formatRichMessage (maybe "" showThreadId -> thread) (maybe "" showTime -> time) Message{..} =
         showSeverity messageSeverity
      <> time
      <> showSourceLoc messageStack
      <> thread
      <> messageText
 
-    showTime :: UTCTime -> Text
-    showTime t = square $ T.pack $
-          formatTime defaultTimeLocale "%H:%M:%S." t
-       ++ take 3 (formatTime defaultTimeLocale "%q" t)
-       ++ formatTime defaultTimeLocale " %e %b %Y %Z" t
+    showTime :: C.Time -> Text
+    showTime t = square $ T.pack $ show $ C.builder_DmyHMS timePrecision datetimeFormat (C.timeToDatetime t)
```

Comment:
Wow, I didn't expect that writing formatter with `chronos` would be that simple! Nice work :+1:

Reply:
Though, using `show` for Builder looks like a performance problem :thinking: I think it's better to use `toLazyText` and `fromLazy` here instead of `show` and `T.pack`:

* https://hackage.haskell.org/package/text-1.2.3.0/docs/Data-Text-Lazy-Builder.html#v:toLazyText

Reply:
I think it's better to use `Data.Text.Lazy.toStrict $ Data.Text.Lazy.Builder.toLazyText`